beard/utils/strings.py

```diff
@@ -13,10 +13,13 @@
 
 """
 
-import unicodedata
+import re
 import sys
+import unicodedata
 
 IS_PYTHON_3 = sys.version_info[0] == 3
```

Comment:
It does not seem to work -- tests dont pass anymore using this. Maybe we can have a look outside this PR. I agree this is an ugly hack I would rather have an clean solution for this.